ui/src/components/TrafficPage.tsx

```diff
@@ -134,23 +135,14 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({setAnalyzeStatus, onTLS
             switch (message.messageType) {
                 case "entry":
                     const entry = message.data;
-                    var focusThis = false;
-                    if (!focusedEntryId) {
-                        focusThis = true;
-                        setFocusedEntryId(entry.id.toString());
+                    if (!focusedEntryId) setFocusedEntryId(entry.id.toString())
+                    let newEntries = [...entries, entry];
```

Comment:
can be const

Reply:
@lirazyehezkel done with https://github.com/up9inc/mizu/commit/5c05b613ab3b9dda63088fadd9c45e39eb844a3d